src/js/actions/index.js

```diff
@@ -424,6 +426,17 @@ export function fetchPurchaseOrderStatuses() {
   };
 }
 
+export function fetchPaymentTerms() {
+  return (dispatch) => {
+    purchaseOrderApi.getPaymentTerms().then((res) => {
```

Comment:
for this simple case it doesn't matter, but IMO async/await convention is generally cleaner